diff -up mess-0.145/makefile.systemlibs mess-0.145/makefile
--- mess-0.145/makefile.systemlibs	2012-02-07 20:16:08.845116674 +0100
+++ mess-0.145/makefile	2012-02-07 20:16:08.849116700 +0100
@@ -211,6 +211,12 @@ BUILD_EXPAT = 1
 # uncomment next line to build zlib as part of MAME build
 BUILD_ZLIB = 1
 
+# uncomment next line to build libjpeg as part of MAME build
+BUILD_JPEG = 1
+
+# uncomment next line to build libflac as part of MAME build
+BUILD_FLAC = 1
+
 # uncomment next line to include the symbols
 # SYMBOLS = 1
 
@@ -411,7 +417,9 @@ DEFS += -DUSE_NETWORK
 endif
 
 # need to ensure FLAC functions are statically linked
+ifeq ($(BUILD_FLAC),1)
 DEFS += -DFLAC__NO_DLL
+endif
 
 
 
@@ -622,14 +630,30 @@ LIBS += -lz
 ZLIB =
 endif
 
+# add jpeg library
+ifeq ($(BUILD_JPEG),1)
+INCPATH += -I$(SRC)/lib/libjpeg
+JPEG_LIB = $(OBJ)/libjpeg.a
+else
+LIBS += -ljpeg
+JPEG_LIB =
+endif
+
+# add flac library
+ifeq ($(BUILD_FLAC),1)
+INCPATH += -I$(SRC)/lib/libflac/include
+FLAC_LIB = $(OBJ)/libflac.a $(OBJ)/libflac++.a
+else
+LIBS += -lFLAC -lFLAC++
+FLAC_LIB =
+endif
+
 # add SoftFloat floating point emulation library
 SOFTFLOAT = $(OBJ)/libsoftfloat.a
 
 # add formats emulation library
 FORMATS_LIB = $(OBJ)/libformats.a
 
-JPEG_LIB = $(OBJ)/libjpeg.a
-
 #-------------------------------------------------
 # 'default' target needs to go here, before the 
 # include files which define additional targets
@@ -639,9 +663,6 @@ default: maketree buildtools emulator
 
 all: default tools
 
-FLAC_LIB = $(OBJ)/libflac.a 
-# $(OBJ)/libflac++.a
-
 
 #-------------------------------------------------
 # defines needed by multiple make files 
diff -up mess-0.145/src/emu/sound/samples.c.systemlibs mess-0.145/src/emu/sound/samples.c
--- mess-0.145/src/emu/sound/samples.c.systemlibs	2012-02-07 20:15:32.260892239 +0100
+++ mess-0.145/src/emu/sound/samples.c	2012-02-07 20:16:08.850116706 +0100
@@ -16,7 +16,11 @@
 #include "emu.h"
 #include "emuopts.h"
 #include "samples.h"
-#include "../../lib/libflac/include/flac/all.h"
+#ifdef FLAC__NO_DLL
+#include <flac/all.h>
+#else
+#include <FLAC/all.h>
+#endif /* FLAC__NO_DLL */
 
 typedef struct _sample_channel sample_channel;
 struct _sample_channel
diff -up mess-0.145/src/lib/util/chd.c.systemlibs mess-0.145/src/lib/util/chd.c
--- mess-0.145/src/lib/util/chd.c.systemlibs	2012-02-07 20:15:23.800840438 +0100
+++ mess-0.145/src/lib/util/chd.c	2012-02-07 20:16:08.854116730 +0100
@@ -48,7 +48,11 @@
 #include <stdlib.h>
 #include <new>
 
-#include "../../lib/libflac/include/flac/all.h"
+#ifdef FLAC__NO_DLL
+#include <flac/all.h>
+#else
+#include <FLAC/all.h>
+#endif /* FLAC__NO_DLL */
 
 
 /***************************************************************************
diff -up mess-0.145/src/mame/drivers/sliver.c.systemlibs mess-0.145/src/mame/drivers/sliver.c
--- mess-0.145/src/mame/drivers/sliver.c.systemlibs	2012-02-07 20:15:57.226045319 +0100
+++ mess-0.145/src/mame/drivers/sliver.c	2012-02-07 20:16:08.856116742 +0100
@@ -67,7 +67,7 @@ Notes:
 #include "sound/okim6295.h"
 #include "cpu/mcs51/mcs51.h"
 #include "video/ramdac.h"
-#include "../../lib/libjpeg/jpeglib.h"
+#include <jpeglib.h>
 
 
 #define FIFO_SIZE 1024
