diff -up mess-0.143/makefile.verbosebuild mess-0.143/makefile
--- mess-0.143/makefile.verbosebuild	2011-06-29 18:37:48.905676677 +0200
+++ mess-0.143/makefile	2011-06-29 18:40:27.154888332 +0200
@@ -297,12 +297,12 @@ BUILD_EXE = $(EXE)
 endif
 
 # compiler, linker and utilities
-AR = @ar
-CC = @gcc
-LD = @g++
+AR = ar
+CC = gcc
+LD = g++
 MD = -mkdir$(EXE)
-RM = @rm -f
-OBJDUMP = @objdump
+RM = rm -f
+OBJDUMP = objdump
 
 
 
@@ -752,12 +752,12 @@ $(OBJ)/%.s: $(SRC)/%.c | $(OSPREBUILD)
 
 $(OBJ)/%.lh: $(SRC)/%.lay $(FILE2STR_TARGET)
 	@echo Converting $<...
-	@$(FILE2STR) $< $@ layout_$(basename $(notdir $<))
+	$(FILE2STR) $< $@ layout_$(basename $(notdir $<))
 
 $(OBJ)/%.fh: $(SRC)/%.png $(PNG2BDC_TARGET) $(FILE2STR_TARGET)
 	@echo Converting $<...
-	@$(PNG2BDC) $< $(OBJ)/temp.bdc
-	@$(FILE2STR) $(OBJ)/temp.bdc $@ font_$(basename $(notdir $<)) UINT8
+	$(PNG2BDC) $< $(OBJ)/temp.bdc
+	$(FILE2STR) $(OBJ)/temp.bdc $@ font_$(basename $(notdir $<)) UINT8
 
 $(DRIVLISTOBJ): $(DRIVLISTSRC)
 	@echo Compiling $<...
